Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -233,12 +233,54 @@ struct PaymentSheetTestPlayground: View {
             }
         }
     }
+    
+    var uiStyleBinding: Binding<PaymentSheetTestPlaygroundSettings.UIStyle> {
+        Binding<PaymentSheetTestPlaygroundSettings.UIStyle> {
+            return playgroundController.settings.uiStyle
+        } set: { newUIStyle in
+            // If we switch to embedded put style back to vertical and default back to deferred CSC if we are in an intent first integration
+            if newUIStyle == .embedded {
+                playgroundController.settings.layout = .vertical
```

Comment:
Even though embedded doesn't use this layout API, I think it makes the most sense to change the toggle to vertical as that's what most resembles embedded.

Reply:
Is it possible to disable the inapplicable toggles rather than changing their selection?

Reply:
Ah yes, easier than I anticipated! I left the logic for changing the confirmation type but now also disable it. I think it makes sense that it always goes back to deferred CSC, then disabling.


![Simulator Screenshot - iOS 18  - 2024-09-24 at 14 32 12](https://github.com/user-attachments/assets/e0dd07ba-7419-4577-a652-32689ee31218)

Reply:
Switching `normal` -> `deferred_csc` makes sense.

IMO I don't want it to switch to `.vertical` though; that's more confusing than helpful since the value is ignored either way. It should just stay whatever value it was.

Reply:
🤦 I agree, I thought I had removed that but clearly didn't. Fixed!